public/app/features/trails/Breakdown/LabelBreakdownScene.tsx

```diff
@@ -338,6 +339,15 @@ export class LabelBreakdownScene extends SceneObjectBase<LabelBreakdownSceneStat
       allLabelOptions.filter((option) => option.value !== ALL_VARIABLE_VALUE).unshift(all);
     }
 
+    const otelWarning = useRef(false);
```

Comment:
This `useRef` is here for "referencing a value that’s not needed for rendering". But can't we use `useState`? Or a variable outside of this whole component and scene? I'd like to discuss this and learn if I am missing something.

Reply:
We can use other means of tracking that the warning has been shown.

Reply:
Updated!